src/components/Navbar/Navbar.scss

```diff
@@ -0,0 +1,38 @@
+.Navbar {
+  width: 100%;
+  height: 50px;
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  top: 0;
```

Comment:
you don't need top here, and font-size is global and will  apply anyway (=> you don't need it here)

flex-direction is row by default (=> you don't need it here)

I would add justify-content: space-between; to place both left and right sides by sides